lib/active_admin/views/components/active_admin_form.rb

```diff
@@ -42,11 +42,13 @@ def build(resource, options = {}, &block)
       end
 
       def inputs(*args, &block)
+        if block_given?
+          form_builder.template.assign('has_many_block'=> true)
```

Comment:
Surrounding space missing for operator '=>'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.